client/tsconfig.json

```diff
@@ -2,6 +2,9 @@
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
     "strict": true,
+    "paths": {
+      "@/*": ["./*"]
+    }
```

Comment:
Is it supposed to include all paths?

Reply:
Yes, for example if you have a path like this ../../../assets/icons/icon.png so this option will replace all the dots and find the relative path of the root folder and will replace it with @ so the path will look like this @/assets/icons/icon.png